README.md

```diff
@@ -400,12 +400,82 @@ end
 
 You normally want to have your coverage analyzed across ALL of your test suites, right?
 
-Simplecov automatically caches coverage results in your (coverage_path)/.resultset.json. Those results will then
-be automatically merged when generating the result, so when coverage is set up properly for Cucumber and your
-unit / functional / integration tests, all of those test suites will be taken into account when building the
-coverage report.
+Simplecov automatically caches coverage results in your (coverage_path)/.resultset.json.
 
-There are two things to note here though:
+### Between subsequent test runs
+
+Test results are be automatically merged with previous runs when generating the result, so when coverage is set up
+properly for Cucumber and your unit / functional / integration tests, all of those test suites will be taken into
+account when building the coverage report.
```

Comment:
`s/Test results are be automatically/Test results are automatically` 

> with previous runs
I think it's when
1. on the same filesystem
2. in SimpleCov.coverage_dir
3. different command_name
4. within SimpleCov.merge_timeout window